design-system/materials/internals/definition.yaml

```diff
@@ -244,6 +262,17 @@ decisionGroupsByTheme:
           description: ''
           choice: color-warning-95
 
+    borders:
+      label: Borders
+      prefix: border
+      decisions:
+        border-for-button-as-secondary:
+          choice: 'none'
```

Comment:
Default theme buttons don't use borders but shadows for border-like visualization.